wiki/troubleshooting.md

```diff
@@ -17,6 +17,14 @@ To run SFC, from am elevated command prompt run `sfc /scannow`. It is recommend
 
 For more information about SFC, check [this page](https://support.microsoft.com/en-us/help/929833/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system)
 
+## CHKDSK
+
+This utility checks the file system and file system metadata of a volume for errors.
```

Comment:
```suggestion
This utility checks the file system and file system metadata of a volume for errors.

```

Reply:
It's identical

Reply:
Line break

Reply:
Oh. Ok

Reply:
I can't find how to acces to the .md to edit it, because GitHub is saying that I can't replace it with an identical text.